From 402b0ca282a737e19562a7c8d28dc6594a48d54c Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Wed, 11 Feb 2015 20:00:50 -0500
Subject: [PATCH 1/3] pkg-rebar: HOST_CC -> HOSTCC

Fix HOSTCC typo that would cause host-rebar builds to fail.

Signed-off-by: Frank Hunleth <fhunleth@troodon-software.com>
---
 package/pkg-rebar.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/pkg-rebar.mk b/package/pkg-rebar.mk
index 620316d..57fb2f4 100644
--- a/package/pkg-rebar.mk
+++ b/package/pkg-rebar.mk
@@ -209,7 +209,7 @@ $(2)_CONF_ENV += $$(REBAR_HOST_DEPS_ENV)
 ifndef $(2)_BUILD_CMDS
 define $(2)_BUILD_CMDS
 	(cd $$(@D); \
-		CC="$$(HOST_CC)" \
+		CC="$$(HOSTCC)" \
 		CFLAGS="$$(HOST_CFLAGS)" \
 		LDFLAGS="$$(HOST_LDFLAGS)" \
 		$$(REBAR_HOST_DEPS_ENV) \
-- 
1.9.1

